features/bootstrap/FeatureContext.php

```diff
@@ -178,6 +178,9 @@ public static function create_cache_dir() {
 	 * @param array $parameters context parameters (set them up through behat.yml)
 	 */
 	public function __construct( array $parameters ) {
+		if ( getenv( 'RESTFUL_TEST_DBHOST' ) ) {
```

Comment:
truetrue. Should then we make the sister code in restful also use WP_CLI_TEST_DBHOST?